rpmbuild/bin/copr-builder

```diff
@@ -167,9 +167,9 @@ def cmd_release():
     """
     Mark this VM as expired
     """
-    expiration = datetime.now() - timedelta(minutes=1)
+    timestamp = 0
     with open(USER_SSH_EXPIRATION_PATH, "w+", encoding="utf-8") as fp:
-        fp.write(str(expiration.timestamp()))
+        fp.write(str(timestamp))
```

Comment:
??? This will always write "0".

Reply:
I suppose it is expected, but I'd be more confident if there was `fp.write("0")` :)

Reply:
That is expected as per the commit message. But I updated the code as @praiskup suggested.